tests/integration/steps_downtime.go

```diff
@@ -186,8 +186,5 @@ func stepsDowntime(consumerName string) []Step {
 				},
 			},
 		},
-
-		// TODO: Test full unbonding functionality, tracked as: https://github.com/cosmos/interchain-security/issues/311
```

Comment:
Is that issue solved now?

Reply:
Not yet. Accidentally deleted the comment.

Comment was reintroduced